src/app/server/BUILD.gn

```diff
@@ -27,6 +27,12 @@ static_library("server") {
   output_name = "libCHIPAppServer"
 
   sources = [
+    "../clusters/scenes/ExtensionFieldSets.h",
+    "../clusters/scenes/ExtensionFieldSetsImpl.cpp",
+    "../clusters/scenes/ExtensionFieldSetsImpl.h",
+    "../clusters/scenes/SceneTable.h",
+    "../clusters/scenes/SceneTableImpl.cpp",
+    "../clusters/scenes/SceneTableImpl.h",
```

Comment:
This should be in clusters/scenes/BUILD.gn and just be a library. You can then put it in the public_deps below. Otherwise, you are duplicating rules in tests/BUILD.gn

Reply:
BUILD.gn added in cluster/scenes.